src/sagemaker/cli/compatibility/v2/modifiers/py_version.py

```diff
@@ -0,0 +1,124 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+"""A class to ensure that ``py_version`` is defined when constructing framework classes."""
+from __future__ import absolute_import
+
+import ast
+
+from sagemaker.cli.compatibility.v2.modifiers.modifier import Modifier
+
+PY_VERSION_ARG = "py_version"
+PY_VERSION_DEFAULT = "py3"
```

Comment:
this should match what the v1 defaults are. specifically, [MXNet](https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/mxnet/estimator.py#L48) and [TensorFlow](https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/tensorflow/estimator.py#L305) originally defaulted to "py2". The latter is a bit more messy now, which could be a reason in favor of combining this with the existing `FrameworkVersionEnforcer` class - if the framework version isn't specified, then TF definitely defaults to "py2"

Reply:
i like your idea. i'll discard this pr and combine the two